src/components/authentication/Login.tsx

```diff
@@ -0,0 +1,105 @@
+// import { useState } from 'react';
+import { useEffect, useState, FormEvent } from 'react';
+import './auth.scss';
```

Comment:
Create a new folder named authentication in styles directory, move all stylesheets there and import for use where necessary.